design_drafts/components_architecture_and_urdf_examples.md

```diff
@@ -367,16 +377,10 @@ Note:
       <classType>ros2_control_components/IMUSensor</classType>
       <stateInterfaceType>velocity</stateInterfaceType>
       <stateInterfaceType>acceleration</stateInterfaceType>
```

Comment:
I would keep the values here as an option.

Reply:
roger that